src/AnalyzeRegistry.jl

```diff
@@ -373,9 +389,19 @@ function analyze(dir::AbstractString; repo = "", reachable=true, subdir="")
         license_files = LicenseTableEltype[]
         lines_of_code = LoCTableEltype[]
     end
+
+    # If the repository is on GitHub and we have a non-anonymous GitHub
+    # authentication, get the list of contributors
+    contributors = if auth isa GitHub.AnonymousAuth || occursin("github.com", repo)
+        repo_name = replace(replace(repo, r"^https://github\.com/" => ""), r"\.git$" => "")
+        String[c["contributor"].login for c in GitHub.contributors(GitHub.repo(repo_name; auth); auth)[1] if c["contributor"].login ∉ ("JuliaTagBot", "github-actions[bot]")]
```

Comment:
Is zeptodoctor one we should exclude too? (If I’m remembering that name right...)

Reply:
Yeah, we need to collect all bots we know, these are the first two I found.